apps/shipments/serializers.py

```diff
@@ -304,14 +307,13 @@ def __init__(self, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
 
-    shipment = ShipmentSerializer(read_only=True)
-
     class Meta:
         model = TrackingData
         fields = '__all__'
 
     def create(self, validated_data):
+        point = Point(validated_data.get('longitude', None), validated_data.get('latitude', None))
```

Comment:
I think this will also be redundant if we move this to a pre_save hook for `TrackingData`